amazons3/amazons3_endpoint.bal

```diff
@@ -40,6 +44,10 @@ public type Client client object {
         string baseURL = HTTPS + amazonS3Config.amazonHost;
         self.accessKeyId = amazonS3Config.accessKeyId;
         self.secretAccessKey = amazonS3Config.secretAccessKey;
+        var token = amazonS3Config.securityToken;
+        if ((token is string) && token != "") {
```

Comment:
Do we need the is empty check? 

We should either just set whatever the user sets, or return an error for empty strings, IMO.

Reply:
yes we need. bcz when we pass root credentials, we don't need to set securityToken in the header. In that case if we haven't set any value for securityToken , the empty value is set. and we can't return error for empty string as well.